Removed unsupported versions from database products (#5764)
EnterpriseDB/docs: product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx

> For more information about using the `pg_ctl` utility, or the command line options available, see the official PostgreSQL Core Documentation available at:

Reviewer: ```suggestion
For more information about using the `pg_ctl` utility, or the command line options available, see the official PostgreSQL Core Documentation available at:
```
```suggestion
For more information about using the `pg_ctl` utility or the command-line options available, see the official [PostgreSQL Core Documentation available](https://www.postgresql.org/docs/11/static/app-pg-ctl.html).
```

> [https://www.postgresql.org/docs/11/static/app-pg-ctl.html]<https://www.postgresql.org/docs/11/static/app-pg-ctl.html>

Reviewer: ```suggestion
```

> For more information about using the command line client, see the PostgreSQL Core Documentation at:

Reviewer: ```suggestion
For more information about using the command line client, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/11/static/app-psql.html).
```

> [https://www.postgresql.org/docs/11/static/app-psql.html]<https://www.postgresql.org/docs/11/static/app-psql.html>

Reviewer: ```suggestion
```

## Configuring a Package Installation

The packages that install the database server component create a unit file (on 8.x hosts) and service startup scripts.

### Creating a Database Cluster and Starting the Service

The PostgreSQL `initdb` command creates a database cluster. After installing Advanced Server, you must manually configure the service and invoke `initdb` to create your cluster. When invoking `initdb`, you can:

-   Specify environment options on the command line.
-   Include the `systemd` service manager on RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.

To review the `initdb` documentation, visit:

> [https://www.postgresql.org/docs/11/static/app-initdb.html]<https://www.postgresql.org/docs/11/static/app-initdb.html>

Reviewer: ```suggestion
```
